src/constants_manager.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#ifndef CONSTANTS_MANAGER_H
+#define CONSTANTS_MANAGER_H
+
+#include "json.h"
```

Comment:
You don't need to include "json.h" - a forward declaration of `JsonObject` is enough.

You also don't need to include "set", but you should include "map" and "string" as the header uses those.